ethcore/src/miner/miner.rs

```diff
@@ -768,6 +769,11 @@ impl miner::MinerService for Miner {
 
 		trace!(target: "own_tx", "Importing transaction: {:?}", pending);
 
+		// Verify RLP payload first
```

Comment:
I didn't put it into RPC dispatcher, because import_own_transaction called not only from RPC. At least 2 other usages (in private transactions and in secret store). IMHO these cases (when transactions are created not from RPC but by some logic) also should be guarded with this check.

Reply:
It should be part of verification pipeline then + the additional check for all external transactions to avoid RLP decoding them and hashing large amounts of data.